modules/ztp-sno-du-configuring-disk-encryption-with-pcr-protection.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-sno-du-configuring-disk-encryption-with-pcr-protection_{context}"]
+= Enabling disk encryption with TPM and PCR protection
+
+You can use the `diskEncryption` field in the `SiteConfig` custom resource (CR) to configure disk encryption with Trusted Platform Module (TPM) and Platform Configuration Registers (PCRs) protection. 
+
+Configuring the `SiteConfig` CR enables disk encryption at cluster installation time.
+
+.Prerequisites
+
+* You have installed the {oc-first}.
+* You have logged in as a user with `cluster-admin` privileges.
+* You read the "About disk encryption with PCR protection" section.
+
+.Procedure
+
+* Configure the `spec.clusters.diskEncryption` field in the `SiteConfig` CR:
```

Comment:
I would recommend making this a two step procedure: `Configure` and `Verify that the disk...`

Reply:
I think keeping the verification step separate will be appropriate because the verification step does not have anything to do with enabling this feature.